lib/vsc/filesystem/gpfs.py

```diff
@@ -221,7 +221,7 @@ def _assemble_fields(self, fields, out):
                     fields[i:i + 1] = map(lambda fs: (len(fs), fs), fixed_lines)
 
         # assemble result
-        listm = Monoid([], lambda xs, ys: xs + ys)  # not exactly the fastest mappend for lists ...
+        listm = MonoidConcat()
```

Comment:
i'm trying to comrehend why we need the monoid here at all. any thoughhts wha regular dict is not enough?

Reply:
That's a good point, I took another look at the code and the only utility of `MonoidDicts` as they are currently used in vsc-filesystems is to 1) have a dict of `Monoid` to automatically append values to lists 2) create any missing keys in the dict on-the-fly.
This can easily be achieved with `collections.defaultdict` and we get even better performance than with my `MonoidConcat`. See the updated description of this PR.